lms/djangoapps/discussion/rest_api/tests/test_api_v2.py

```diff
@@ -1055,639 +1110,3 @@ def test_vote_count_two_users(
                 vote_count -= 1
                 assert result["vote_count"] == vote_count
                 self.register_get_user_response(self.user, upvoted_ids=[])
-
-
-@ddt.ddt
-@mock.patch.dict("django.conf.settings.FEATURES", {"ENABLE_DISCUSSION_SERVICE": True})
-class GetThreadListTest(
```

Comment:
I'm confused by this. Why is this being removed? Weren't these tests added here [just a month ago](https://github.com/openedx/edx-platform/commit/e0fbb96ee7ad390d0cfbd3235e69fbbee3d5ec9e#diff-e1e11c053bdb5c3b5419d659da606d685f5a40a05aa7f24b87ffbd2188495e90)? It also seems to be removing things that are unrelated to subscriptions?

Reply:
Thank you for pointing this out. I didn't notice this as I rebased the changes, and it removes this in the last PR. I'll fix this soon.